apps/sms/test/marionette/inbox_test.js

```diff
@@ -76,4 +76,69 @@ marionette('Inbox View tests', function() {
       });
     });
   });
+
+  suite('Split Inbox view tests', function() {
+    var numberOfConversations = 10;
+    var numberOfMessagesInConversation = 5;
```

Comment:
nit: I'd use capital letters, to better indicate they're constants. Don't know what you think ? Also maybe we can use "const" now that we use node v4 ?